controls/stig_rhel9/SRG-OS-000163-GPOS-00072.yml

```diff
@@ -2,13 +2,18 @@ controls:
     -   id: SRG-OS-000163-GPOS-00072
         levels:
             - medium
-        title: 'The operating system must terminate all network connections associated with
+        title: '{{{ full_name }}} must terminate all network connections associated with
     a communications session at the end of the session, or as follows: for in-band
     management sessions (privileged sessions), the session must be terminated after
-    10 minutes of inacti'
+    10 minutes of inactivity; and for user sessions (non-privileged session),
+    the session must be terminated after 15 minutes of inactivity,
+    except to fulfill documented and validated mission requirements.'
         rules:
-            - sshd_set_idle_timeout
-            - sshd_set_keepalive
-            - sshd_set_keepalive_0
+            #- sshd_set_idle_timeout
+            #- sshd_set_keepalive
+            #- sshd_set_keepalive_0
             - accounts_tmout
-        status: automated
+        # We don't have a way to configure ssh timeout on RHEL 9, the
+        # sshd_config options ClientAliveInterval and ClientAliveCountMax
+        # dont't work on RHEL 9 as they used to in RHEL 8.
+        status: partial
```

Comment:
@ggbecker @Mab879
 As we already know SSH newer than 8.2 doesn't allow to configure the timeout and there is no solution for the time being. How to track this problem in the control file?

Reply:
@ggbecker @Mab879 Do you have any suggestions? Or is the way proposed in this PR fine for you?

Reply:
So the SSHD lines do provide some values, mainly around resource exhaustion. However, they do not provide, you noted, the functionality provided by RHEL 8.5 and below. We don't have good answer right now. If we want to merge this PR, in the current state I believe that "does not meet" best describes what is going on. The Mitigation column could mention `TMOUT` and tmux locking.